### DISCUSSION TITLE
Example code using undeclared function initialize on the BlockingScheduler

### DISCUSSION THREAD

**Discussion by RaemondBW** [agronholm/apscheduler#646] [Q&A] [answered]
Hi I was trying this library out and found that the examples didn't run because of their usage of `scheduler.initialize()` vs `scheduler.start()`? I don't see anyone else having the same problem in the issues, so maybe there is a problem with my setup?

`AttributeError: 'BlockingScheduler' object has no attribute 'initialize'`

from the example here: https://github.com/agronholm/apscheduler/blob/master/examples/executors/processpool.py

**Comment by agronholm** [comment, marked as answer]
I suggest you look at the [3.x branch](https://github.com/agronholm/apscheduler/blob/3.x/examples/executors/processpool.py), as master reflects the still work-in-progress 4.0 version.
